lib/helpers/filterJsDocComments.js

```diff
@@ -0,0 +1,26 @@
+const jsYaml = require('js-yaml');
+
+/**
+ * Filters JSDoc comments for those tagged with '@swagger'
+ * @function
+ * @param {array} jsDocComments - JSDoc comments
+ * @returns {array} JSDoc comments tagged with '@swagger'
+ * @requires js-yaml
+ */
+function filterJsDocComments(jsDocComments) {
+  const swaggerJsDocComments = [];
+
+  for (let i = 0; i < jsDocComments.length; i += 1) {
```

Comment:
IMHO You should use `for ... of` loop, it's more readdable

Reply:
I agree, there are many other places where the codebase should be improved! Again, didn't refactor too much in order to focus on the issue at hand...